app/src/main/java/com/better/alarm/presenter/AlarmsListActivity.kt

```diff
@@ -202,6 +208,28 @@ class AlarmsListActivity : AppCompatActivity() {
             }.apply { }
     }
 
+    private fun showMuteAlert() {
```

Comment:
This function does something conditionally. This should be mentioned in the function name. A short comment would also be helpful, but it is not necessary.